python/katana/cpp/libgalois/graphs/Graph.pxd

```diff
@@ -20,6 +20,9 @@ cdef extern from "tsuba/RDG.h" namespace "tsuba" nogil:
         optional[vector[string]] node_properties
         optional[vector[string]] edge_properties
 
+cdef extern from "rdg_generated.h" namespace "katana::fbs" nogil:
+    cdef struct RDGAccessT:
+        pass
```

Comment:
Add a TODO to add fields here and expose them to python as a named tuple. There isn't much value in having these if we cannot use them from Python.